test/unit/model/delivery/container/LtiDeliveryContainerTest.php

```diff
@@ -0,0 +1,152 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA
+ *
+ */
+
+namespace oat\taoLtiConsumer\test\unit\model\delivery\container;
+
+use core_kernel_classes_Resource as RdfResource;
+use IMSGlobal\LTI\ToolProvider\ToolConsumer;
+use oat\generis\model\data\Model;
+use oat\generis\test\unit\OntologyMockTest;
+use oat\oatbox\session\SessionService;
+use oat\oatbox\user\User;
+use oat\tao\model\oauth\DataStore;
+use oat\taoDelivery\model\execution\DeliveryExecution;
+use oat\taoLtiConsumer\model\delivery\container\LtiDeliveryContainer;
+use oat\taoLtiConsumer\model\delivery\container\LtiExecutionContainer;
+use phpmock\MockBuilder;
+use PHPUnit_Framework_MockObject_MockObject as MockObject;
+use Zend\ServiceManager\ServiceLocatorInterface;
+
+class LtiDeliveryContainerTest extends OntologyMockTest
+{
+    public function testGetExecutionContainer()
+    {
+        $resourceLinkId = 'id of the resource';
```

Comment:
maybe we can split this because is hard to read

Reply:
It is big indeed, as is the tested method as you mentioned.
Nevertheless, I'm not sure how to split it and if I have time for it.